README.adoc

```diff
@@ -192,8 +221,28 @@ server.xml
 include::finish/src/main/liberty/config/server.xml[tags=**;!comment]
 ----
 
+== Building and running the application
+
+To build the application, run the Maven `install` phase from the command line in the `start` directory:
+[role='command']
+```
+mvn install
+```
+
+This command builds the application and creates a `.war` file in the target directory. It also
+configures and installs Open Liberty into the `target/liberty/wlp` directory.
+
+Next, run the Maven `liberty:start-server` goal:
 [role='command']
-include::{common-includes}/mvnbuild.adoc[]
+```
+mvn liberty:start-server
+```
+
+This goal starts an Open Liberty server instance. Your Maven `pom.xml` is already configured to start
+the application in this server instance.
+
+Navigate your browser to this URL to access the application: https://localhost:9080/ServletSample/servlet[https://localhost:9080/ServletSample/servlet^]. The application displays a simple web page with the link that, when 
```

Comment:
I think this syntax `https://localhost:9080/ServletSample/servlet[^]` works for adding the URL with opening on a new tab action.